src/routes/contact.js

```diff
@@ -0,0 +1,232 @@
+ * tags:
+ *  name: Contact
+ *  description: Api for contact
+ * /api/contact:
+ *   get:
+ *     summary: Get all contact request
+ *     tags: [Contact]
+ *     responses:
+ *       200:
+ *         description: The list of all contact request
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: array
+ *               items:
+ *                 $ref: '#/components/schemas/Contact'
+ *       500:
+ *         description: Internal server error
+ *   post:
+ *     summary: Create a new contact request
+ *     tags: [Contact]
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             required:
+ *              - name
+ *              - category
+ *              - description
+ *              - email
+ *             type: object
+ *             properties:
+ *               name:
+ *                 type: string
+ *               category:
+ *                 type: string
+ *               description:
+ *                 type: string
+ *               email:
+ *                 type: string
+ *     responses:
+ *       200:
+ *         description: Contact request successfully created
+ *       400:
+ *         description: Missing argument(s)
+ *       500:
+ *         description: Internal server error
+ *
+ * /api/contact/{uuid}:
+ *   patch:
+ *     summary: Update a contact request
+ *     tags: [Contact]
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             required:
+ *               - name
+ *               - email
+ *               - processed
+ *             type: object
+ *             properties:
+ *               name:
+ *                 type: string
+ *               email:
+ *                 type: string
+ *               processed:
+ *                 type: boolean
+ *     parameters:
+ *       - name: uuid
+ *         in: path
+ *         description: Uuid of the request
+ *         required: true
+ *         items:
+ *           type: string
+ *     responses:
+ *       200:
+ *         description: Contact request successfully updated
+ *       400:
+ *         description: Missing argument(s)
+ *       404:
+ *         description: Contact not found (wrong uuid)
+ *       500:
+ *         description: Internal server error
+ *   delete:
+ *     summary: Delete a contact request
+ *     tags: [Contact]
+ *     parameters:
+ *       - name: uuid
+ *         in: path
+ *         description: Uuid of the request
+ *         required: true
+ *         items:
+ *           type: string
+ *     responses:
+ *       200:
+ *         description: Contact request successfully deleted
+ *       400:
+ *         description: Missing argument(s) (uuid)
+ *       404:
+ *         description: Contact not found (wrong uuid)
+ *       500:
+ *         description: Internal server error
+ */
+
+contact_router.post('/', async (req, res) => {
```

Comment:
You don't need to be authenticated to use the routes ?

Reply:
not for the POST but GET PATCH and DELETE yes i'll change it